src/fdisk.ts

```diff
@@ -0,0 +1,140 @@
+const completionSpec: Fig.Spec = {
+  name: "fdisk",
+  description: "Manipulate disk partition table",
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for fdisk",
+    },
+    {
+      name: ["--version", "-V"],
+      description: "Show version for lsblk",
+    },
+    {
+      name: ["--sector-size", "-b"],
+      description: "Specify the sector size of the disk",
+      args: {
+        name: "mode",
+        description: "Valid values are 512, 1024, 2048, and 4096",
+      },
+    },
+    {
+      name: ["--protect-boot", "-B"],
+      description:
+        "Don't erase the beginning of the first disk sector when creating a new disk label",
+    },
+    {
+      name: ["--compatibility", "-c"],
+      description: "Specify the compatibility mode, 'dos' or 'nondos'",
+      args: {
+        name: "mode",
+        isOptional: true,
+      },
```

Comment:
Can't help but read `nondos` as a typo of Nando's 🫠
```suggestion
      args: {
        name: "mode",
        isOptional: true,
        suggestions: ["dos", "nondos"],
      },
```